core/zms/src/main/rdl/Domain.rdli

```diff
@@ -319,4 +319,20 @@ resource AuthHistoryDependencies GET "/domain/{domainName}/history/auth" {
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
+}
+
+type ExpiredMembers Struct {
+    Array<MemberName> expiredRoleMembers; //list of deleted expired role members names
+    Array<GroupMemberName> expiredGroupMembers; //list of deleted expired groups members names
+}
+
+// Delete expired principals
+resource ExpiredMembers DELETE "/expired-members?purgeResources={purgeResources}" {
```

Comment:
the definition of ExpiredMembers looks incomplete. We're just returning a list of names but that's not sufficient. What roles where those users members of? We need to specify the role/domain names and maybe even the expiration date.

I see we have defined PurgeMember in the code. Why not define it in the RDL maybe with a more generic name and include the expiration field as well

Reply:
My thought was, a list of strings would look as follows: <domain_name>:<collection_name>:<member_name>
However, it might be better to return PurgeMember instead of String.